upup/pkg/fi/fitasks/keypair.go

```diff
@@ -240,6 +248,17 @@ func (_ *Keypair) Render(c *fi.Context, a, e, changes *Keypair) error {
 		klog.Infof("updated Keypair %q to new format", name)
 	}
 
+	// Load the keypair for dependent resources
+	{
+		cert, _, _, err := c.Keystore.FindKeypair(name)
```

Comment:
You _could_ optimize this out...

Reply:
This may have been an attempted workaround for the problem that was actually solved by IsReady in the other PR (now).  It doesn't seem to be needed yet, so I removed from this PR.